libs/wingsdk/src/target-sim/api.inflight.ts

```diff
@@ -217,3 +223,21 @@ function transformRoutePath(route: string): string {
   // route validation is done in the preflight file
   return route.replace(/{/g, ":").replace(/}/g, "");
 }
+
+// express v4 doesn't natively handle async request handlers, so we need to
+// wrap them in a middleware function
+function asyncMiddleware(
```

Comment:
The only change in this file was adding this wrapper to resolve an eslint warning from a new rule I added